src/features/session_trace/aggregate/index.js

```diff
@@ -27,15 +29,17 @@ const toAggregate = {
   mousing: [1000, 2000],
   touching: [1000, 2000]
 }
-const MAX_TRACE_DURATION = 15 * 60 * 1000 // 15 minutes
+const MAX_TRACE_DURATION = 10 * 60 * 1000 // 10 minutes
+const REQ_THRESHOLD_TO_SEND = 30
 
 export class Aggregate extends AggregateBase {
   static featureName = FEATURE_NAME
   constructor (agentIdentifier, aggregator, argsObj) {
     super(agentIdentifier, aggregator, FEATURE_NAME)
+    this.agentRuntime = getRuntime(agentIdentifier)
```

Comment:
This should probably be moved to the `FeatureBase`. Better yet, the agent instance should be passed down into the instrument and aggregate classes upon initialization so we don't have to use utility functions like this. But I digress.

Reply:
Would go in its own PR to switch all the feature modules' getRuntime -> this.agentRuntime.

Reply:
I agree. Something to discuss internally.